new-lamassu-admin/src/pages/Triggers/Triggers.js

```diff
@@ -30,82 +26,71 @@ const SAVE_CONFIG = gql`
-  const { data, loading } = useQuery(GET_INFO)
-  const triggers = fromServer(data?.config?.triggers ?? [])
+  const { data, loading } = useQuery(GET_CONFIG)
+  const { data: customInfoReqData } = useQuery(GET_CUSTOM_REQUESTS)
+
+  const customInfoRequests =
+    R.path(['customInfoRequests'])(customInfoReqData) ?? []
+  const enabledCustomInfoRequests = R.filter(R.propEq('enabled', true))(
+    customInfoRequests
+  )
 
+  const triggers = fromServer(data?.config?.triggers ?? [])
   const complianceConfig =
     data?.config && fromNamespace('compliance')(data.config)
   const rejectAddressReuse = complianceConfig?.rejectAddressReuse ?? false
-  const [error, setError] = useState(null)
 
   const [saveConfig] = useMutation(SAVE_CONFIG, {
     onCompleted: () => setWizard(false),
-    refetchQueries: () => ['getData'],
-    onError: error => setError(error)
+    refetchQueries: () => ['getData']
   })
 
-  const add = rawConfig => {
-    const toSave = R.concat([{ id: v4(), direction: 'both', ...rawConfig }])(
-      triggers
-    )
-    return saveConfig({ variables: { config: { triggers: toServer(toSave) } } })
-  }
-
   const addressReuseSave = rawConfig => {
     const config = toNamespace('compliance')(rawConfig)
     return saveConfig({ variables: { config } })
   }
 
-  const save = config => {
-    setError(null)
-    return saveConfig({
-      variables: { config: { triggers: toServer(config.triggers) } }
-    })
-  }
-
-  const currency = R.path(['fiatCurrency'])(
-    fromNamespace(namespaces.LOCALE)(data?.config)
-  )
-
   const titleSectionWidth = {
     [classes.tableWidth]: !showCustomInfoRequests
   }
 
-  const [customRequests, setCustomRequests] = useState([])
-
-  const alter = R.curry((values, key, items) => {
-    const merge = R.mergeLeft(values)
-    return R.map(R.when(R.propEq('id', key), merge), items)
-  })
-
-  const handleDeleteCustomRequest = req => {
-    return setCustomRequests(R.reject(o => o.id === req.id)(customRequests))
+  const setBlur = shouldBlur => {
+    return shouldBlur
+      ? document.querySelector('#root').classList.add('root-blur')
+      : document.querySelector('#root').classList.remove('root-blur')
   }
 
-  const handleSaveCustomRequest = (values, isEditing) => {
-    if (!isEditing) {
-      return setCustomRequests([
-        ...customRequests,
-        { id: customRequests.length, ...values }
-      ])
+  const toggleWizard = wizardName => forceDisable => {
+    if (forceDisable) {
+      setBlur(false)
+      return setWizard(null)
     }
-    return setCustomRequests(alter(values, values.id, customRequests))
-  }
-
-  const toggleCustomRequestWizard = () => {
-    document.querySelector('#root').classList.toggle('root-blur')
-    const wizardOpen = !!wizardType
-    return wizardOpen ? setWizard(false) : setWizard('newCustomRequest')
+    if (wizardType === wizardName) {
+      setBlur(false)
+      return setWizard(null)
+    }
```

Comment:
These two if statements use the same code for different conditions

`if (wizardType === wizardName || forceDisable)`